flowauth/frontend/src/Token.js

```diff
@@ -33,7 +54,12 @@ class Token extends React.Component {
           <Typography component="p">{expiry}</Typography>
         </Grid>
         <Grid item xs={3}>
-          <Button onClick={this.toggleOpen}>Token</Button>
+
+          <Tooltip title={copySuccess} placement="bottom">
+            <Button type='button' onClick={this.copyToClipboard} id="copy" color="primary">Copy</Button>
```

Comment:
Can we make these outlines buttons instead? https://material-ui.com/demos/buttons/#alternatives